tools/publish/processing.js

```diff
@@ -419,8 +419,9 @@ function doCompleteIDL(conf, page, n) {
     utils.forEachNode(doc, function(n) {
       if (n.nodeType == n.ELEMENT_NODE &&
           n.localName == "pre" &&
-          n.getAttribute("class") == "idl") {
-        if (n.classList.contains("extract")) {
+          /\bidl\b/.test(n.getAttribute("class")) ) {
```

Comment:
We probably can use `.className` here instead.

Reply:
I don't think xmldom exposes `className` (unless they've added it and not changed their docs).

As you mentioned elsewhere, xmldom might not be the most ideal tool to use, but I'm trying to minimize changes to the tools. When we do change the SVG build system, it will to be replace it with a well-maintained alternative like Bikeshed.